fe/fe-core/src/main/java/org/apache/doris/catalog/TableIndexes.java

```diff
@@ -56,6 +58,26 @@ public TableIndexes(List<Index> indexes, Map<String, String> properties) {
         this.properties = properties;
     }
 
+    private synchronized int incAndGetMaxIndexUniqueId() {
```

Comment:
Oh, there is still a problem to use `maxIndexId` inside `TableIndexes`.
For example, after `add index`, there will be an edit log, and after FE restart, it will replay that edit log, but it didn't update the `maxIndexId` in `TableIndexes`.

Reply:
change to Env.getCurrentEnv().getNextId()